tools/icqsol_add_texture/icqsol_add_texture.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+import argparse
+import shutil
+from pkg_resources import resource_filename
```

Comment:
@pletzer This complexity needs to be encapsulated in the Shape Manager - similar to not import bumpy into Galaxy tool wrappers, underlying components like resource_filename should not be imported here either.  See below.